.github/workflows/end-to-end-test-portals.yml

```diff
@@ -0,0 +1,90 @@
+name: End to end testing for Portals
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+env:
+  PW_ALL_BLOBS_DIR: all-blob-reports
+  PORTALS_SUBDIR: apps/portals
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  e2e-tests:
+    if: ${{ (github.event_name == 'push' && (github.ref_name == 'main' || github.actor == github.repository_owner)) || github.event_name == 'pull_request' }}
+    strategy:
+      fail-fast: false
+      max-parallel: ${{ github.repository_owner == 'Sage-Bionetworks' && 5 || 10 }}
```

Comment:
Limit number of parallel e2e jobs in the Sage repo

Reply:
You could consider adding a comment to why 5 were used.